src/modules/sandbox/components/SideNav.tsx

```diff
@@ -1,66 +1,72 @@
 import React from 'react';
 import { Box, Grid, Stack, StxInline, StackProps, color } from '@stacks/ui';
 import { border } from '@common/utils';
-import { ClarityIcon } from '@sandbox/components/clarity-icon';
-import { Tooltip } from '@components/tooltip';
+import { ClarityIcon } from '@modules/sandbox/components/ClarityIcon';
 import FunctionIcon from 'mdi-react/FunctionIcon';
 import { DropIcon } from '@components/icons/drop';
-import { Routes } from '@sandbox/common/types';
 import Link from 'next/link';
-import { useConnect } from '@sandbox/hooks/use-connect';
 import { useAppSelector } from '@common/state/hooks';
 import { selectActiveNetwork } from '@common/state/network-slice';
 import { buildUrl } from '@components/links';
+import { useRouter } from 'next/router';
+import { useUser } from '@modules/sandbox/hooks/useUser';
 
 export const SideNav: React.FC<StackProps> = props => {
-  const { isSignedIn } = useConnect();
   const networkMode = useAppSelector(selectActiveNetwork).mode;
+  const router = useRouter();
+  const { isConnected } = useUser();
 
   const navigation: {
     label: string;
     url: string;
     icon: any;
     isDisabled?: boolean;
+    isSelected: boolean;
   }[] = [
     {
       label: 'Write & Deploy Contracts',
       url: buildUrl(`/sandbox/deploy`, networkMode),
       icon: <ClarityIcon size="24px" />,
+      isSelected: router.pathname === '/sandbox/deploy',
```

Comment:
I'm quite sure this is the case, but did you confirm that `router.pathname` does _not_ include the query string if it's present?

Reply:
yep it doesn't include query string